src/perform_step/rkc_perform_step.jl

```diff
@@ -370,7 +370,8 @@ end
   @unpack t, dt, uprev, u, f, p, fsalfirst = integrator
   maxeig!(integrator, cache)
   # The the number of degree for Chebyshev polynomial
-  maxm = max(2,Int(floor(sqrt(integrator.opts.internalnorm(integrator.opts.reltol,t)/(10*eps(integrator.opts.internalnorm(uprev,t)))))))
+  #maxm = max(2,Int(floor(sqrt(integrator.opts.internalnorm(integrator.opts.reltol,t)/(10*eps(integrator.opts.internalnorm(uprev,t)))))))
```

Comment:
what's this all about?

Reply:
`maxm` here could be a huge number, so I am setting it to 50 for now.